### ISSUE TITLE
Deprecated warning on PHP 8.1

### ISSUE THREAD

**Issue by lcharette** [JackieDo/Laravel-Dotenv-Editor#45]
https://github.com/JackieDo/Laravel-Dotenv-Editor/blob/f93690a80915d51552931d9406d79b312da226b9/src/Jackiedo/DotenvEditor/DotenvReader.php#L138

The line above will return the following warning on PHP 8.1 : 

```
PHP Deprecated:  auto_detect_line_endings is deprecated in {...}/jackiedo/dotenv-editor/src/Jackiedo/DotenvEditor/DotenvReader.php on line 138
```

https://wiki.php.net/rfc/deprecations_php_8_1#auto_detect_line_endings_ini_setting

**Comment by jontjs** [comment]
I'm also getting these:

> Deprecation Notice: strlen(): Passing null to parameter #1 ($string) of type string is deprecated in /var/www/html/britishhamper2021/vendor/jackiedo/dotenv-editor/src/Jackiedo/DotenvEditor/DotenvFormatter.php:59
> Deprecation Notice: trim(): Passing null to parameter #1 ($string) of type string is deprecated in /var/www/html/britishhamper2021/vendor/jackiedo/dotenv-editor/src/Jackiedo/DotenvEditor/DotenvFormatter.php:79

For background, see https://php.watch/versions/8.1/internal-func-non-nullable-null-deprecation & https://wiki.php.net/rfc/deprecate_null_to_scalar_internal_arg

**Comment by JackieDo** [comment]
@lcharette, @jontjs:
This issue has been resolved (commit 154228bacd1f3eddc38371cb309c45e0e0d3f071) and included in the [1.2.1](https://github.com/JackieDo/Laravel-Dotenv-Editor/releases/tag/1.2.1) release
